users_ldap_mail/__init__.py

```diff
@@ -19,4 +19,5 @@
 #
 ##############################################################################
 
-from . import users_ldap_model
+from . import models
```

Comment:
It would be great if you could replace the old large copyright statement with the new leaner one.